src/datakit/main.ml

```diff
@@ -41,65 +41,47 @@ let error fmt = Printf.ksprintf (fun s ->
 
 let max_chunk_size = Int32.of_int (100 * 1024)
 
-let make_task msg =
+let info msg =
   let date = Int64.of_float (Unix.gettimeofday ()) in
-  Irmin.Task.create ~date ~owner:"datakit <datakit@docker.com>" msg
-
-(* FIXME: this is a bit ridiculous *)
-module Contents_string = struct
-  open Irmin.Contents.String
-  type t = string
-  let equal = equal
-  let compare = compare
-  let hash = hash
-  let to_json = to_json
-  let of_json = of_json
-  let size_of = size_of
-  let write = write
-  let read = read
-  let merge _ = merge []
-  module Path = Ivfs_tree.Path
-end
+  Irmin.Info.v ~date ~author:"datakit <datakit@docker.com>" msg
```

Comment:
Email is mandatory, so Irmin will add the default `irmin@openmirage.org` email to it. But yes `@docker.com` should go away.